cypress/integration/ut2019.spec.js

```diff
@@ -1,14 +1,55 @@
-describe('ut2019', function() {
-	const email = 'test.user@ut2019.ee';
-	const name = 'Test User';
-	const message = 'Custom message!';
-	it('Submit form successfully', function() {
-		cy.clearCookies();
-		cy.visit('https://ut2019.voog.com/contact');
-		cy.get('.contact-name input').type(name);
-		cy.get('.contact-email input').type(email);
-		cy.get('.contact-message textarea').type(message);
-		cy.get('input[type="submit"]').click();
-		cy.get('.form_notice').should('have.text', 'Thank you!');
-	});
-});
+//List of different language options
+describe('generated from a list', () => {
+    const operations = [{
+            value: "et",
+        },
+        {
+            value: "en",
+        },
+    ]
+
+    operations.forEach((operation) => {
+
+        describe("ut2019", function() {
+            const email = "test.user@ut2019.ee";
+            const name = "Test User";
+            const message = "Custom message!";
+            it("Submit form successfully", function() {
+                cy.clearCookies();
+                cy.visit('https://ut2019.voog.com/' + operation.value);
```

Comment:
You could use the base url https://ut2019.voog.com as a configurable variable and not duplicate it across the tests. In a real-world scenario you most probably will run these tests against different environments and it will help you to easily re-configure your tests against dev/test environments.